boa/src/builtins/map/mod.rs

```diff
@@ -66,7 +65,7 @@ impl BuiltIn for Map {
         )
         .property(
             to_string_tag,
-            "Map",
+            Self::NAME,
```

Comment:
For Json and Math we use the type name rather than `Self`, I'm happy with either one, but we should choose one.
```suggestion
            Map::NAME,
```

Reply:
But, `Self::NAME` is equivalent to `Map::NAME` here, right? since we are in the `Map` impl. Also, if we ever decide to change the name of `Map`, this would automatically continue working.

Reply:
I think this will be consistent after #1276.

Reply:
Sounds good